### ISSUE TITLE
fx:anySlot in property paths breaks fx:properties 

### ISSUE THREAD

**Issue by enridaga** [SPARQL-Anything/sparql.anything#114]
Not sure why, but a query using `fx:anySlot` in a property path does not return content and spits the following error message:
```
ERROR 2021-10-05 21:14:32,543 [main] (FacadeXOpExecutor.java:343) - Neither location nor content provided
```
Will create a test case to reproduce.

**Comment by enridaga** [comment]
The same query works when putting the location in the service URI.

**Comment by luigi-asprino** [comment]
Closed via 540f1f4
